README.md

```diff
@@ -0,0 +1,76 @@
+Readtime plugin for pelican
+===================================
+
+Plugin for [Pelican](http://getpelican.com/) that computes read time based on [Medium's readtime](https://help.medium.com/hc/en-us/articles/214991667-Read-time)
+"algorithm".
+
+It adds a ``readtime`` and ``readtime_string`` attributes to every Articles and/or Pages, 
+with the time estimation for reading the article.
+
+## Setting Up
+
+#### Adding 'ReadTime' to the list of plugins:
+```python
+PLUGINS = [ 
+    ... , 
+    'readtime'
+]
+```
+
+#### 1. Words Per Minute Only
+
+In your settings you would use assign the `READTIME_LANGUAGE_SUPPORT` variable to an integer like so:
+
+***pelicanconf.py***
+```python
+READTIME_LANGUAGE_SUPPORT = 180
```

Comment:
`READTIME_LANGUAGE_SUPPORT` seems a bit ambiguous, maybe something like `READTIME_WPM`?

Reply:
I have changed according to your suggestion, I will commit ASAP ;)